### ISSUE TITLE
every time you save is added a pair of parentheses

### ISSUE THREAD

**Issue by dobbbri** [ThisIsManta/vscode-stylus-supremacy#24]
```

font-family = -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-
serif
font-weight = {
  light: 300,
  normal: 400,
  bold: 700
}
base = {
  bg: white,
  color: slate,
  disabled-color: grey-30,
  line-height: 1.7,
  font-weight: ((((((((font-weight.normal)))))))),
  font-size: 1rem,
  font-family: font-family
}


@media (max-width: (((((((grid-breakpoints.mobile))))))))
  h1
    font-size: round(base.font-size * (typography.small-size-ratio ** 6), 3)
```

**Comment by dobbbri** [comment]
how i fix this?

**Comment by ThisIsManta** [comment]
Ah, I didn't have a chance to look at this yet. Please bear with me.

**Comment by rossity** [comment]
@ThisIsManta Any update on this? It's still happening when using objects for instance:

```
@media (min-width: $sizes['xs'])
    padding 20px
```

or 
```
@media (min-width: $sizes.xs])
    padding 20px
```

It adds more parentheses every time you save leading to something like this after a couple of saves:
```
@media (min-width: ((((((($sizes['xs']))))))))
    padding 20px
```

edit: @sldobri my temporary fix is to give my objects temporary variables
```
$sizes = {
  xs: 320px,
  sm: 375px,
  md: 768px,
  lg: 1024px,
  xl: 1920px
}
$screen-xs = $sizes['xs']
$screen-sm = $sizes['sm']
$screen-md = $sizes['md']
$screen-lg = $sizes['lg']
$screen-xl = $sizes['xl']

.container
  @media (min-width: $screen-xs)
    padding 1em
```

**Comment by gaven** [comment]
I'm having a very similar issue. Any word on a fix?

<img width="547" alt="screen shot 2018-07-16 at 5 15 32 pm" src="https://user-images.githubusercontent.com/11202081/42789785-021e4038-891c-11e8-8d24-7f0abbab303c.png">

**Comment by ThisIsManta** [comment]
Working on this 😜

**Comment by ThisIsManta** [comment]
@sldobri, I could not replicate `font-weight: ((((((((font-weight.normal)))))))),`. Would you kindly double check?

**Comment by ThisIsManta** [comment]
`@media (min-width: ((((((($sizes['xs']))))))))` has been fixed at version [2.12.1](https://github.com/ThisIsManta/stylus-supremacy/releases/tag/v2.12.1).

Cheers!
